Document/0x06e-Testing-Cryptography.md

```diff
@@ -75,18 +68,18 @@ Mobile SDKs offer standard implementations of RNG algorithms that produce number
 
 #### Static Analysis
 
-Apple provides developers with the Randomisation Services application programming interface (API) that generates cryptographically secure random numbers<sup>[1]</sup>.
+Apple provides developers with the Randomisation Services application programming interface (API) that generates [cryptographically secure random numbers](https://developer.apple.com/reference/security/randomization_services "Randomization Services").
 
-The Randomisation Services API uses the `SecRandomCopyBytes` function to perform the numbers generation. This is a wrapper function for the <code>/dev/random</code> device file, which provides cryptographically secure pseudorandom value from 0 to 255 and performs concatenation<sup>[2]</sup>. 
+The [Randomisation Services API](https://developer.apple.com/library/content/documentation/Security/Conceptual/cryptoservices/RandomNumberGenerationAPIs/RandomNumberGenerationAPIs.html "Generating Random Numbers") uses the `SecRandomCopyBytes` function to perform the numbers generation. This is a wrapper function for the <code>/dev/random</code> device file, which provides cryptographically secure pseudorandom value from 0 to 255 and performs concatenation.
```

Comment:
Good catch. I removed the link, since the text above references the correct page, while the text below references the specific function.